Cargo.toml

```diff
@@ -40,6 +28,7 @@ overflow-checks = true
 [workspace.dependencies]
 #workspace members
 attestation = { path = "crates/attestation" }
+data-transfer-objects = { path = "crates/data_transfer_objects" }
```

Comment:
This crate is dedicated for the contract interface afaik, so I'd like the name of the crate to reflect that. Perhaps `contract-dtos` or `contract-interface-types`. This is basically a first step towards https://github.com/near/mpc/issues/515 as well.